utils_linux.go

```diff
@@ -81,6 +81,8 @@ func newProcess(p specs.Process) (*libcontainer.Process, error) {
 		Label:           p.SelinuxLabel,
 		NoNewPrivileges: &p.NoNewPrivileges,
 		AppArmorProfile: p.ApparmorProfile,
+		ConsoleWidth:    uint16(p.ConsoleSize.Width),
+		ConsoleHeight:   uint16(p.ConsoleSize.Height),
```

Comment:
This will silently take the lowest 16 bits of the specified value, which could be confusing. I think it is better to pass and store as `int` everywhere and error on console creation if it is too big (in future other platforms could have different limits).